### ISSUE TITLE
Paketti Pitchbend Sample Loader menu added to right-click menu in Renoise's Sample Loader

### ISSUE THREAD

**Issue by tkna91** [esaruoho/org.lackluster.Paketti.xrnx#39]
It would be useful to be able to run Paketti's Pitchbend Sample Load in the file chooser in Renoise.

![20240428-234833_screenshot](https://github.com/esaruoho/org.lackluster.Paketti.xrnx/assets/102382754/ba0ec9e0-6ecc-4ca6-ac20-b68c808a47d7)

![20240428-234738_screenshot](https://github.com/esaruoho/org.lackluster.Paketti.xrnx/assets/102382754/4def5e0c-3098-4f79-9f81-194f5b9c4080)

**Comment by esaruoho** [comment]
good idea. it is very trivial to add these to anywhere. (pattern editor, instrument box, sample editor, etc..)


<img width="718" alt="Screenshot 2024-04-28 at 22 46 21" src="https://github.com/esaruoho/org.lackluster.Paketti.xrnx/assets/4966687/01245f23-72e7-47c1-906d-bcba7f295879">
[org.lackluster.Paketti_V3.54.xrnx.zip](https://github.com/esaruoho/org.lackluster.Paketti.xrnx/files/15143482/org.lackluster.Paketti_V3.54.xrnx.zip)

please verify that it works as expected and close the ticket when you're satisfied.

**Comment by tkna91** [comment]
Since Renoise's file chooser is already open, I think the process should be done without opening a new file chooser.

https://github.com/esaruoho/org.lackluster.Paketti.xrnx/assets/102382754/81717bad-9659-46b8-97f1-6e430b013fb6

**Comment by tkna91** [comment]
The idea is that it would be useful to be able to load the `*st_Pitchbend Instrument` into multiple or single slots before loading the (multiple) selected samples in the “Samples” area of Renoise.
This is a totally different feature in that sense, since it does not go through the OS standard file chooser.

**Comment by esaruoho** [comment]
@tkna91 ok, sounds interesting, please open a separate ticket about that and give me examples in the description. thanks!
